subsys/bluetooth/common/Kconfig

```diff
@@ -94,20 +94,37 @@ config BT_BUF_ACL_RX_COUNT
 	  When Controller to Host flow control is not enabled the Controller
 	  can assume that the Host has infinite amount of buffers.
 
+config BT_BUF_CMD_COMPLETE_EVT_SIZE
+	  int "Maximum supported HCI Command Complete Event buffer length"
+	  # LE Read Supported Commands command complete event.
+	  default 68
+	  range 68 255
```

Comment:
The values here could possible be optimized a bit. 

From what I can tell, the largest SIG return parameters is 31 octets (some ISO test event), and most of them is below 10 octets. 

The default value can probably in most cases be around 10 octets, and possible up to 20 in some cases. I think it makes sense to set the default value of the command complete event to be as low as possible depending on what has been enabled. 

The range can probably be lowered as well, as we never need a minimum size of 68. Technically the minimum can be down to 3 octets, right, for commands without return parameters. The minimum should probably also depend on what has been enabled. 

Reducing the size of this would also mean that `BT_BUF_EVT_RX_SIZE` probably not default to this

Reply:
https://github.com/zephyrproject-rtos/zephyr/pull/74645/files#diff-b199a522335b761f8d05548154babf2722b207853be7e7e02cfa8eb21913bf9aR99

<img width="1919" alt="image" src="https://github.com/zephyrproject-rtos/zephyr/assets/26378606/ac0de868-80ed-44c7-b083-5f526d1140d4">

Reply:
Ah, thanks. I only went through the LE controller commands, so missed this. Thanks!